$NetBSD: patch-meson.build,v 1.2 2020/03/04 15:11:49 wiz Exp $

Turn into a meson.build file which meson will accept, and
subsitute some variables.

--- meson.build.orig	2020-02-27 16:12:52.000000000 +0000
+++ meson.build
@@ -1,3 +1,5 @@
+project('gdbus_codegen')
+
 gdbus_codegen_files = [
   '__init__.py',
   'codegen.py',
@@ -9,11 +11,11 @@ gdbus_codegen_files = [
 ]
 
 gdbus_codegen_conf = configuration_data()
-gdbus_codegen_conf.set('VERSION', glib_version)
-gdbus_codegen_conf.set('MAJOR_VERSION', major_version)
-gdbus_codegen_conf.set('MINOR_VERSION', minor_version)
-gdbus_codegen_conf.set('PYTHON', python_name)
-gdbus_codegen_conf.set('DATADIR', glib_datadir)
+gdbus_codegen_conf.set('VERSION', '@VERSION@')
+gdbus_codegen_conf.set('MAJOR_VERSION', '@MAJOR_VERSION@')
+gdbus_codegen_conf.set('MINOR_VERSION', '@MINOR_VERSION@')
+gdbus_codegen_conf.set('PYTHON', '@PYTHON@')
+gdbus_codegen_conf.set('DATADIR', '@DATADIR@')
 
 # Install gdbus-codegen executable
 gdbus_codegen = configure_file(input : 'gdbus-codegen.in',
@@ -24,7 +26,7 @@ gdbus_codegen = configure_file(input : '
   # Provide tools for others when we're a subproject and they use the Meson GNOME module
 meson.override_find_program('gdbus-codegen', gdbus_codegen)
 
-codegen_dir = join_paths(glib_datadir, 'glib-2.0', 'codegen')
+codegen_dir = join_paths('@DATADIR@', 'glib-2.0', 'codegen')
 
 gdbus_codegen_built_files = []
 gdbus_codegen_built_files += configure_file(input : 'config.py.in',
